acceptance/tests/pxp-module-puppet/rerun_transaction.rb

```diff
@@ -0,0 +1,60 @@
+require 'pxp-agent/test_helper.rb'
+
+
+STATUS_QUERY_MAX_RETRIES = 60
+STATUS_QUERY_INTERVAL_SECONDS = 1
+
+test_name 'two runs with same transaction_id' do
```

Comment:
Acceptance tests need to be tracked in Test Rail - I created test case C99777 for this new test. Can the test_name string be updated to 'C99777 - two runs with same transaction_id' please? 
If everything else is done with this PR; go ahead and merge and I'll update this later